resources/css/tag.css

```diff
@@ -1,13 +1,38 @@
 .tag {
-  display: inline-block;
-  opacity: .9;
+  display: inline-flex;
+  align-items: center;
+  opacity: 0.9;
   margin: 0 .15em;
```

Comment:
>and the margin here removed as it can be just an inline element implicitly 

Main reason margin is there is to make the tags more spaced out. They look a bit too close to text without it, IMO.

>I think the achievement-set classes are not needed anymore at this point so could be removed altogether? The margin of .achievement-set.title should not be applied globally.

Indeed, considering how barren `achievement-set.css` looks now, that class is just a leftover at this point. I think some game avatars could use a larger gap between icon and title, but there are better ways to solve that.

>If variants of tags should exist they should be just tag variants themselves?

Makes the most sense to me. They can generally just go to `tag.css`.

Reply:
Added the margin back 👍